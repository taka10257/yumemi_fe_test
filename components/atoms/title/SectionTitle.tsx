import React from 'react'
import styles from './SectionTitle.module.sass'

type SectionTitleProps = {
  title: string
}

export default function SectionTitle({ title }: SectionTitleProps) {
  return (
    <div className={styles.root}>
      <h2 className={styles.title}>{title}</h2>
    </div>
  )
}
