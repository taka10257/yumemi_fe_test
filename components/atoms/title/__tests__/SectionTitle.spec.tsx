import React from 'react'
import { shallow } from 'enzyme'
import { expect } from '@jest/globals'
import SectionTitle from '../SectionTitle'

describe('components/atoms/title/SectionTitle', () => {
  describe('正常系', () => {
    it('初期表示', () => {
      const header = shallow(<SectionTitle title={'タイトル'} />)
      expect(header).toMatchSnapshot()
    })
  })
})
