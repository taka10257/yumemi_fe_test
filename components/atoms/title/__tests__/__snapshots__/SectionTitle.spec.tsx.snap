// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/title/SectionTitle 正常系 初期表示 1`] = `
<div
  className="root"
>
  <h2
    className="title"
  >
    タイトル
  </h2>
</div>
`;
